frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -36,6 +37,22 @@ const tableColumnClasses = [
   Kebab.columnClass,
 ];
 
+const NodeResource = {
```

Comment:
`NodeResource` is defined at 2 places. Better to define at one and use it from there.